src/styles/styles.js

```diff
@@ -1550,6 +1550,12 @@ const styles = (theme) => ({
         justifyContent: 'flex-start',
         paddingBottom: 16,
     },
+    chatContentScrollViewWithHeaderLoader: {
+        // regular paddingBottom wouldn't work here
```

Comment:
Please explain why 😄

Reply:
There were conflicts with the parent styles. I tried it again, and it works well.